README.md

```diff
@@ -10,9 +10,9 @@ custom elements.
 polydev must be built before running. The build step externalizes inline scripts
 for CSP compliance, and copies some dependencies into convenient locations.
 
-To build, run `gulp`:
+To build, run `npm run build`:
```

Comment:
the big reason here is
https://docs.npmjs.com/misc/scripts#path

So a user does not have to have gulp installed globally, and rather will use the version that we have specified in node_modules.

For people that work across many projects, with many versions of gulp/grunt/node/iojs depending on a single global install is quite error prone :crying_cat_face: